src/shared/containers/VideoPluginContainer.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import { Resource } from '@bbp/nexus-sdk';
+import { useNexusContext } from '@bbp/react-nexus';
+import ReactPlayer from 'react-player';
+import * as moment from 'moment';
+import { Collapse, Modal, Button, List, Avatar } from 'antd';
+
+import '../styles/video-plugin.less';
+
+const { Panel } = Collapse;
+
+type VideoProps = {
+  orgLabel: string;
+  projectLabel: string;
+  resource: Resource;
+};
+
+type VideoObject = {
+  name: String;
```

Comment:
is there a difference between `String` and `string`? can we use just `string` to be consistent? :)